.travis.yml

```diff
@@ -37,7 +37,6 @@ script:
 - cabal new-build
 - cabal new-run semantic:test
 - cabal new-run semantic-core:test
-- cabal new-run semantic-core:doctest
```

Comment:
1. With `Analysis.Concrete` moved into `semantic-analysis`, we no longer have any doctests in `semantic-core`.

2. The doctest in `Analysis.Concrete` involved `semantic-core`’s `eval` & `Core`, and didn’t really belong there; it was quite low value so I’ve 🔥d it for the time being, and will replace it with tests for `semantic-analysis` in future PR(s).